tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/deid/motor_vehicle_accident_suppression_test.py

```diff
@@ -9,8 +9,6 @@
 # Python Imports
 import os
 
-from google.cloud.bigquery import Table
```

Comment:
is this change supposed to be in this PR?

Reply:
Yes. I only removed that unused import to trigger integration.